root/views/comments.jade

```diff
@@ -6,20 +6,19 @@ block extrahead
     script(src='/js/comments_module.js')
 
 block content
-    h1 Please leave a comment
-    div(ng-app='commentApp', ng-controller='commentController')
-      div(ng-repeat='one_comment in all_comments')
-        p {{one_comment.name}}: {{one_comment.thought}}
-        time(am-time-ago="one_comment.last_edited")
-        div(ng-hide="one_comment.isuser === false")
-          a.editLink(heref="#") Edit
-          .editControls
-            form
-              input(type='text',ng-model="editForm[one_comment.textboxname]")
-              button(ng-click='editComment(one_comment._id)',type='submit') Save
-          button(ng-click='deleteComment(one_comment._id)') Delete
-        br
-      br
-      form
+    div(ng-app='commentApp', ng-controller='commentController' class='comment-section')
```

Comment:
Basically the only thing I changed here was making the buttons into links with class button